### PR TITLE
chore(userspace/libscap): cleanup some libscap includes and defines

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

/area libscap

**Does this PR require a change in the driver versions?**

NONE

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by FedeDP** [comment]
This should help us fix a build issue on non-linux platforms (even if libs CI is ok :/ )

File: userspace/libscap/CMakeLists.txt [hunk 1]
```diff
@@ -188,9 +188,11 @@ add_subdirectory(engine/udig)
...
 
 if(CMAKE_SYSTEM_NAME MATCHES "Linux")
-	add_definitions(-DHAS_ENGINE_BPF)
-	add_subdirectory(engine/bpf)
-	target_link_libraries(scap scap_engine_bpf)
+	if (NOT MINIMAL_BUILD)
+		add_definitions(-DHAS_ENGINE_BPF)
+		add_subdirectory(engine/bpf)
+		target_link_libraries(scap scap_engine_bpf)
+	endif()
```
Comment (gnosek) [inline userspace/libscap/CMakeLists.txt:195]: ❤️

**Review by gnosek — APPROVED** [review]

**Review by Andreagit97 — APPROVED** [review]
/approve

**Pull request merged**